Tests iOS/MyList/ArchiveFiltersTests.swift

```diff
@@ -41,6 +41,7 @@ class ArchiveFiltersTests: XCTestCase {
         }
 
         try server.start()
+        UIPasteboard.general.string = ""
```

Comment:
it seems the banner was appearing because there was a copy of `https://getpocket.com/` in the clipboard, so I nil out the pasteboard so it doesn't break these tests. open to better ideas

Reply:
The pasteboard containing a URL is almost certainly due to another test that is messing with the pasteboard. I took a look around some other tests and it could be [RecommendationViewModelTests](https://github.com/Pocket/pocket-ios/blob/3b3cb4b846660ae73b89ed3640dfa8df578ffaa1/PocketKit/Tests/PocketKitTests/RecommendationViewModelTests.swift#L306).

That test should clear out UIPasteboard in its own `tearDown` method so we don't have to do that here.

(not sure if this is the exact test that causing the issue, there could be others)

Reply:
thanks! I cleared out the pasteboard in the teardown for `RecommendationViewModelTests` and `SavedItemViewModelTests` instead